test/ApiTestCase.php

```diff
@@ -1232,12 +1232,12 @@ private function createPodcastEpisodeJson(int $number, bool $isSnippet = false,
             unset($podcastEpisode['impactStatement']);
             unset($podcastEpisode['subjects']);
             unset($podcastEpisode['chapters'][0]['impactStatement']);
-            unset($podcastEpisode['content']);
+            unset($podcastEpisode['chapters'][0]['content']);
         }
 
         if ($isSnippet) {
             unset($podcastEpisode['image']['banner']);
-            unset($podcastEpisode['content']);
+            unset($podcastEpisode['chapters'][0]['content']);
```

Comment:
according to https://github.com/elifesciences/api-raml/blob/develop/src/snippets/podcast-episode.v1.yaml a Podcast snippet should have no chapters at all, hence `unset($podcastEpisode['chapters']);`?